client/src/views/invitations/InvitationsPage.vue

```diff
@@ -0,0 +1,923 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page class="invitations-page">
+    <div
+      v-if="isSmallDisplay"
+      class="switch-view"
+      @click="openInvitationSwitchModal"
```

Comment:
I'd put the `click` action in the `ion-button` instead, right now we can open the modal by clicking around the button.